docs/data/date-pickers/lifecycle/lifecycle.md

```diff
@@ -51,9 +51,12 @@ In all the below scenarios, the picker closes when `onClose` is called, except i
 #### When the last view is completed
 
 When a selection in the last view is made, `onClose` will be called only if the `closeOnSelect` prop is equal to `true`.
-By default, it is set to `true` on desktop and `false` on mobile.
+The default value of `closeOnSelect` depends on the component:
 
-Here are a few examples:
+- Date Picker and Date Range Picker: `true` on desktop and `false` on mobile variants;
+- Time Picker, Date Time Picker, and Date Time Range Picker: `false` on desktop and mobile variants.
```

Comment:
```suggestion
- Time Picker, Date Time Picker and Date Time Range Picker: `false` on desktop and mobile variants.
```

Reply:
These were incorrect suggestions.
Time Picker, Date Time Picker and Date Time Range Picker: `false` on desktop and mobile variants.
![Screenshot 2024-12-30 at 16 55 04](https://github.com/user-attachments/assets/58960e1c-4679-4408-8012-6e333ab749c5)

Reply:
https://www.grammarly.com/blog/punctuation-capitalization/comma-before-and/

This seems to be a complex topic :laughing:

Reply:
Ok... I didn't read that blog post before...
It seems that both variants are correct, but I think that we mostly use `,` before the "3rd" `and`.